python/gilded_rose.py

```diff
@@ -5,35 +5,53 @@ class GildedRose(object):
     def __init__(self, items):
         self.items = items
 
+    def is_sulfuras(self, item):
+        return "Sulfuras" in item.name
+
+    def is_conjured(self, item):
+        return "Conjured" in item.name
+
+    def is_backstage_passes(self, item):
+        return "Backstage passes" in item.name
```

Comment:
`Conjured` and `Backstage passes` are only at the beginning of the item. `Something conjured something` isn't a Conjured item 😄